docs/UserGuide.md

```diff
@@ -171,23 +171,33 @@ VirusTracker data saves in the hard disk automatically after any command that ch
 
 ### Adding a visit: 
 
-Adds a visit by the personId, location of visit and date of visit
+Adds a visit by the personId, locationId and date of visit.
 
-Format: `add personId location date`
+Format: `addVisit PersonId LocationId d/DATE`
 
-* The visit is added to the visits list to track close contacts, especially for the infected people
-* PersonId refers to the id stored in the people list which is available before using this app
-* location refers to the particular location the person with the personId visits
-* date refers to the particular date the person has visited the location
+* The visit is added to the visits list to track close contacts, especially for the infected people.
+* PersonId refers to the id stored in the people list.
+* LocationId refers to the id stored in the location list that the person with the personId visits.
```

Comment:
Following Siang Ern's comment, referencing the index rather than id is required for this command. Also, "personId" and "locationId" are hiden away from the user. Users by right should not know about the id.

Reply:
sry just saw this..